scan/git_test.go

```diff
@@ -0,0 +1,330 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package scan
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"gotest.tools/v3/assert"
+	is "gotest.tools/v3/assert/cmp"
+)
+
+func TestParseRemoteURL(t *testing.T) {
```

Comment:
Thanks for the unit test. Can you also add some testcases for devops? You can some url example from url_test.go

https://github.com/Azure/acr-builder/blob/627fa8df6a16b8742f7a55daf90415a6dce100e6/util/url_test.go#L22

Reply:
Added